pandas/tests/window/test_rolling.py

```diff
@@ -130,29 +130,94 @@ def test_closed_fixed(closed, arithmetic_win_operators):
     df_fixed = DataFrame({"A": [0, 1, 2, 3, 4]})
     df_time = DataFrame({"A": [0, 1, 2, 3, 4]}, index=date_range("2020", periods=5))
 
-    result = getattr(df_fixed.rolling(2, closed=closed, min_periods=1), func_name)()
-    expected = getattr(df_time.rolling("2D", closed=closed), func_name)().reset_index(
-        drop=True
-    )
+    result = getattr(
```

Comment:
why is this needed? center is default False

Reply:
I removed this redundand "False". :)